.github/configs/go-release-drafter.yml

```diff
@@ -0,0 +1,27 @@
+include-paths:
+  - "go/"
+name-template: "go-v$RESOLVED_VERSION"
```

Comment:
@jamesL92 In reference to your comment about versioning with GitHub, will these tags serve as the versions? Also, could you clarify how a user can pin a specific version? Thanks!

Reply:
@jamesL92 I think the README of the go dir is a good place for now, we are working on a dedicated docs site for the OSS alone, when it be ready I will let you know.